services/project-grafana-loki/metadata.yaml

```diff
@@ -2,6 +2,10 @@ displayName: Grafana Loki
 description: Horizontally-scalable, highly-available, multi-tenant log aggregation system inspired by Prometheus. Requires Logging Operator.
 category:
   - logging
+dependencies:
+  - project-logging
+  - grafana-loki
```

Comment:
I'm not sure if this requires `grafana-loki` 🤔 

* `grafana-loki` is the workspace deployment of Loki (which shouldn't affect this app). or maybe there was another reason this was here?
* `project-logging` configures the log flows/outputs for fluentd/fluentbit so logs are pushed to Loki... which also requires the `logging-operator`. so both of those look good!
* This app also uses `rook-ceph` and `rook-ceph-cluster`, but it could be configured to not use them.

Reply:
We "require" users to deploy the ws level logging stack as a prereq to deploying the project logging stack, which is probably why those are defined here https://docs.d2iq.com/dkp/2.4/multi-tenant-logging-overview

Reply:
ah gotcha, in that case we can keep `grafana-loki` here